### DISCUSSION TITLE
dagre 布局如何设置居中呀

### DISCUSSION THREAD

**Discussion by xuoutput** [antvis/G6#4094] [Q&A]
<img width="1157" alt="image" src="https://user-images.githubusercontent.com/16385443/203935939-a6671bb2-a940-4a97-90bb-33779b5ec123.png">
就是把这个图 放在画布中间, 左右居中, 或上下居中的配置需要怎么设置呀

**Comment by CorneliusYang** [comment]
align: '', 这样就好
